analyzers/src/SonarAnalyzer.Common/Facade/ILanguageFacade.cs

```diff
@@ -44,3 +44,23 @@ public interface ILanguageFacade<TSyntaxKind> : ILanguageFacade
     ISyntaxKindFacade<TSyntaxKind> SyntaxKind { get; }
     ITrackerFacade<TSyntaxKind> Tracker { get; }
 }
+
+public static class LanguageFacadeExtensions
```

Comment:
The concept of extensions of LangaugeFacade doesn't feel right. It's like dependencies going in both directions. 

Why not change just `is RegexOptions` to `is int` and then cast it?

Reply:
I added (and rebased) a set of whitespaces that are fine in regexes.